Sources/SwiftKuery/QueryBuilder.swift

```diff
@@ -43,6 +43,8 @@ public class QueryBuilder {
         case booleanFalse
         /// The ALL function for subqueries.
         case all
+        /// character used to quote identifiers (table name,column name, etc) with spaces or special characters
+        case identifierQuoteCharacter
```

Comment:
Please change to: 
"Character used to quote identifiers (table name, column name, etc.) with spaces or special characters."
(start sentence with uppercase, spaces, periods).